File: 4SIS.IOS.MeusDados/Meus Dados/ViewController.m
//
//  ViewController.m
//  Meus Dados
//
//  Created by Usuário Convidado on 05/02/2018.
//  Copyright © 2018 Victor Rodrigues. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    // enquanto o APP carrega, execute os códigos abaixo
    self.labelNome.text = @"Meu nome é: ";
    self.labelCidade.text = @"Minha cidade é: ";
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


//evento de click do botao
- (IBAction)btnExibir:(id)sender {
    
    //quando o botao for clicado, alterar os labels apra os valores abaixo
    self.labelNome.text = @"Victor";
    self.labelCidade.text = @"São Paulo";
}
@end

File: 4SIS.IOS.UIKIT/4SIS.IOS.UIKIT/4SIS.IOS.UIKIT/ViewController.swift
//
//  ViewController.swift
//  4SIS.IOS.UIKIT
//
//  Created by Usuário Convidado on 09/04/2018.
//  Copyright © 2018 Victor. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var meuSegmento: UISegmentedControl!
    @IBOutlet weak var meuStepper: UIStepper!
    @IBOutlet weak var lblAno: UILabel!
    @IBOutlet weak var lblKm: UILabel!
    @IBOutlet weak var lblSms: UILabel!
    
    @IBAction func indexChanged(_ sender: Any) {
        switch meuSegmento.selectedSegmentIndex {
        case 0:
            print("Você selecionou o 1º botaão")
            break
        case 1:
            print("Você selecionou o 2º botaão")
            break
        case 2:
            print("Você selecionou o \(String(describing: meuSegmento.titleForSegment(at: meuSegmento.selectedSegmentIndex)))")
            break
        default:
            print("")
        }
    }
    
    @IBAction func stepperValueChanged(_ sender: Any) {
        lblAno.text = String(Int(meuStepper.value))
    }
    override func viewDidLoad() {
        super.viewDidLoad()

        lblAno.text = "1980"
        lblKm.text = "7500"
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

